config/transform-elasticsearch/documents/datadiscovery_study.json

```diff
@@ -41,14 +41,57 @@
         }
       ]
     },
+    "entryType": {
+      "{list}": [
+        {
+          "{or}": [
+            {
+              "{replace}":{
+                "possible_terms": [
+                  "Genotyping", "Genotyping Study",
+                  "allele size", "genotype"
+                ]
+              },
+              "{with}": {
+                "replaced_by": "Genotyping Study"
+              }
+            },
+            {
+              "{replace}":{
+                "possible_terms": [
+                  "Phenotypes", "Phenotyping", "Field Experiement",
+                  "Greenhouse (29\u00baC/20\u00baC)", "Green house",
```

Comment:
Is this character really correspond to this one: http://www.amp-what.com/unicode/search/%5Cu00ba ?

Reply:
Don't you mean this one: http://www.amp-what.com/unicode/search/%5Cu00B0 ?

Reply:
Same as above